README.md

```diff
@@ -34,689 +34,86 @@ npm install --save react-redux-firebase
 
 [react-redux-firebase](https://github.com/prescottprue/react-redux-firebase) provides [`withFirestore`](http://react-redux-firebase.com/docs/api/withFirestore.html) and [`firestoreConnect`](http://react-redux-firebase.com/docs/api/firestoreConnect.html) higher order components, which handle automatically calling `redux-firestore` internally based on component's lifecycle (i.e. mounting/un-mounting)
 
-## Use
```

Comment:
Was about to merge, but was hoping to make this a minor (since there there are already alpha versions of v1 with proposed changes). Because of that we we should probably keep this normal use section for folks which aren't using the cache reducer